benchmarks/helics/ActionMessageBenchmarks.cpp

```diff
@@ -10,138 +10,138 @@ SPDX-License-Identifier: BSD-3-Clause
 
 using namespace helics;
 
-static void BM_AM_toString (benchmark::State &state)
+static void BM_AM_toString(benchmark::State & state)
```

Comment:
Is the space here after '&' desirable?

Reply:
I think with `&` my preference is similar to `*`, attached. Maybe slightly more so with `&`.